pkgs/desktops/deepin/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, pkgs, libsForQt5 }:
+let
+  functions = with pkgs.lib; rec {
+    getPatchFrom' = commonRp:
+      let
+        rpstr = a: b: " --replace \"${a}\" \"${b}\"";
```

Comment:
Don't use cryptic names. This is not C code.

Reply:
This functions not used yet, I will adjust this function in the future pr
This function has been removed in this pr